layouts/partials/data-science.html

```diff
@@ -8,27 +8,44 @@
         </div>
         <div>
             <p>
-                Data Science makes it possible to analyze massive amounts of data
-                and gain meaningful insights. A typical data science workflow involves
-                various techniques and tools such as:
+                NumPy lies at the core of a rich ecosystem of data science libraries.
+            </p>
+            <p>
+                Data science is the analysis of massive amounts of data
+                to gain insight. A typical workflow might be:
+
                 <ul class="content-tab">
-                    <li><b>Extract, Transform, Load (ETL):</b> Pandas, Beautiful Soup, Intake</li>
-                    <li><b>Explore:</b> Seaborn, Matplotlib</li>
-                    <li><b>Model:</b> Scikit-learn, SciPy, statsmodels</li>
-                    <li><b>Evaluate:</b> NumPy, TensorFlow </li>
+                    <li><b>Extract, Transform, Load (ETL):</b>
+                        <a href="https://pandas.pydata.org">Pandas</a>,
+                        <a href="https://www.crummy.com/software/BeautifulSoup/">Beautiful Soup</a>,
+                        <a href="https://intake.readthedocs.io/en/latest/"> Intake</a>
+                    </li>
+
+                    <li><b>Explore:</b>
+                        <a href="https://seaborn.pydata.org"> Seaborn</a>,
+                        <a href="https://matplotlib.org">Matplotlib</a>,
+
+                    </li>
+
+                    <li><b>Model:</b>
+                        <a href="https://scikit-learn.org">scikit-learn</a>,
+                        <a href="https://www.scipy.org">SciPy</a>,
+                        <a href="https://www.statsmodels.org/stable/index.html"> statsmodels</a>.
+                    </li>
+
+                    <li><b>Evaluate:</b>
+                        NumPy,
+                        <a href="https://www.tensorflow.org">TensorFlow</a>
+                    </li>
+
                     <li>
-                        <b>Presentation:</b>
+                        <b>Display:</b>
                         <a href="./index.html/#tab-visual"> Data Visualization Tools</a>
```

Comment:
New issue found: this doesn't work as well as it should; it opens the correct tab, but the view jumps back to top of the page. Noting here, can deal with it later.